Tests/API/ApiTests.cs

```diff
@@ -596,6 +596,47 @@ public void BacktestingData_CanBeDownloadedAndSaved_Successfully()
             Assert.IsTrue(File.Exists(minutePath));
         }
 
+        /// <summary>
+        /// Test read price API for given symbol
+        /// </summary>
+        [Test]
+        public void ReadPriceWorksCorrectlyFor1Symbol()
+        {
+            var spy = Symbol.Create("SPY", SecurityType.Equity, Market.USA);
+            var pricesList = _api.ReadPrices(new [] { spy });
+
+            Assert.IsTrue(pricesList.Success);
+            Assert.AreEqual(pricesList.Prices.Count, 1);
+
+            var price = pricesList.Prices.First();
+            Assert.AreEqual(price.Symbol, spy.ID.ToString());
+            Assert.AreNotEqual(price.Price, 0);
+            var updated = price.Updated;
+            var reference = DateTime.UtcNow.Subtract(TimeSpan.FromDays(3));
+            Assert.IsTrue(updated > reference);
```

Comment:
interesting assert condition, these are integrations tests so it's all gravy

Reply:
Yeah, my thought was it should pass even if you run it on a Sunday